### PR TITLE
Add comprehensive guides and improve documentation 

### PR DESCRIPTION
**Pull request by Flo0807**
The guides should now cover in detail what Backpex can do and therefore make Backpex more beginner-friendly 🚀

### CODE REVIEW

File: guides/get_started/installation.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  @impl Backpex.LiveResource
+  def fields do
+  [
+    title: %{
+      module: Backpex.Fields.Text,
+      label: "Title"
+    },
+    views: %{
+      module: Backpex.Fields.Number,
+      label: "First Name"
```
Comment (krns) [inline guides/get_started/installation.md:180]: Maybe you meant Likes or something else?

Comment (Flo0807) [reply guides/get_started/installation.md:180]: yes, probably a copy & paste mistake. see https://github.com/naymspace/backpex/pull/342

File: guides/actions/item-actions.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+# Item Actions
+
+An item action defines an action (such as deleting a user) that can be performed on one or more items. Unlike resource actions, item actions are not automatically performed on all items in a resource.
+
+An item action could be something like deleting a user, or sending an email to a specific user.
+
+There are multiple ways to perform an Item Action:
+- use the checkboxes in the first column of the resource table to select 1-n items and trigger the action later on
+- use an icon in the last column of the resource table to perform the Item Action for one item
+- use the corresponding icon in the show view to perform the Item Action for the corresponding item
```
[the remark this answers is not in the document]
Comment (pehbehbeh) [reply guides/actions/item-actions.md:10]: I don't think so. 🤔

[the remark this answers is not in the document]
Comment (krns) [reply guides/actions/item-actions.md:10]: @Flo0807 didn't you say that the item actions in show views were buggy?

[the remark this answers is not in the document]
Comment (Flo0807) [reply guides/actions/item-actions.md:10]: yes, I've created an issue: https://github.com/naymspace/backpex/issues/381

**Pull request merged**